test/scalability/lib.py

```diff
@@ -21,9 +21,17 @@ def plotit(csv_file_name: str) -> None:
     fig, (summary_subplot, details_subplot) = plt.subplots(2, 1)
     for i, target in enumerate(targets):
         target_sha = target.split(" ")[1].strip("()")
-        target_comment = subprocess.check_output(
-            ["git", "log", "-1", "--pretty=format:%s", target_sha], text=True
-        )
+
+        target_comment = None
+        try:
+            target_comment = subprocess.check_output(
+                ["git", "log", "-1", "--pretty=format:%s", target_sha], text=True
+            )
+        except subprocess.CalledProcessError:
+            # Sometimes mz_version() will report a Git SHA that is not available
```

Comment:
Should we log this? (optional)

Reply:
Yes, it is visible in the Jupyter notebook when it happens -- the original error from git log is shown in the notebook. I am still at a loss why it happens so frequently, so will need to figure out an alternative way to obtain the git comment going forward.